site/content/sign-in-with-livechat/sdk-documentation.md

```diff
@@ -27,8 +27,42 @@ This method initiates the SDK and returns the `accountsSdk` object instance. It
 
 * **onIdentityFetched(error, data)** – the callback executed when user's identity is fetched. The callback will include either `error` or `data` object depending on the current user authorization status.<br>You will find the detailed documentation in <a href="#response-format">Response format</a> section.
 
+* **linkLabel** – allows to change default _Sign in with LiveChat_ label with custom text.
+
+* **popupRoute** – allows to change default popup's endpoint, to i.e. `signup`.
```

Comment:
It would be nice to have a list of possible values or at least a link to a proper one.